reference/auth/models/palmtreerequest.v1.yaml

```diff
@@ -0,0 +1,8 @@
+title: Palm Tree Request Data
+description: Data to send to Palm Tree's API.
+type: object
+properties:
+  csr:
+    description: Base64 encoded certificate signing request.
```

Comment:
If the details matter... The Coastal request data indicates that it is "base64 encoded DER certificates".

Here, it says only "Base64 encoded"... Is it like Coastal, in that it's base64 DER, or rather, is it PEM (which is also base64 encoded, but with a slightly alternative structure).

Not a big deal to call it out, but you could save someone a bit of head scratching.